python/sparkdl/transformers/tf_tensor.py

```diff
@@ -137,12 +98,9 @@ def _transform(self, dataset):
             out_tnsr_op_names = [tfx.op_name(tnsr_name) for tnsr_name, _ in output_mapping]
             # Load graph
             tf.import_graph_def(graph_def=graph_def, name='', return_elements=out_tnsr_op_names)
-
             # Feed dict maps from placeholder name to DF column name
-            feed_dict = {self._getSparkDlOpName(
-                tnsr_name): col_name for col_name, tnsr_name in input_mapping}
+            feed_dict = {tfx.op_name(tnsr_name): col_name for col_name, tnsr_name in input_mapping}
```

Comment:
Thanks for the review! Agree the `op_name` function is kinda confusing - in this PR we are just using it to strip the ":" part of the tensor name. Having a utility for getting the name of an op seems useful e.g. in case TensorFlow changes their op/tensor naming scheme, but I'm also open to not using it.

I think it'd be nice to:

* Simplify the `op_name` function so that it accepts just a tensor name instead of one of {tensor, tensor name, op name} - AFAICT this is possible without losing any existing functionality, but if not we could break `op_name` up into multiple utils.
* Rename usages of `tfx` -> `graph_utils` for clarity

For now maybe we can use `tfx.op_name` & eliminate all usages of `op_name` in a future PR?